app/renderer/components/Inspector/Source.js

```diff
@@ -77,14 +77,17 @@ class Source extends Component {
     const treeData = source && recursive(source);
 
     return <div id='sourceContainer' className={InspectorStyles['tree-container']}>
-      {source &&
+      {treeData ?
```

Comment:
can we add a comment here to explain what we're doing? it's strange enough we might want to leave a note for our future selves or developers, something like `// the antd Tree component doesn't handle changes in treeData well, so it's better to switch to a new component when we have treedata` (or whatever it is that you think is actually the issue, I could have worded it wrongly)

Reply:
Will do!